modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/NumberUtil.java

```diff
@@ -22,13 +23,13 @@
 public class NumberUtil {
 
 	public static int getDecimalSeparatorIndex(String value) {
-		int index = value.indexOf(StringPool.PERIOD);
-
-		if (index == -1) {
-			index = value.indexOf(StringPool.COMMA);
+		for (int i = 0; i < value.length(); i++) {
+			if (_DECIMAL_SEPARATORS.indexOf(value.charAt(i)) >= 0) {
+				return i;
+			}
```

Comment:
I assume you chose this approach for better performance, but I didn't get it at first sight and, since this isn't a performance hogger, I feel it would be more intuitive if we went with something like:
```suggestion
		for (String decimalSeparator : _DECIMAL_SEPARATORS) {
			int index = value.indexOf(decimalSeparator);

			if (index >= 0) {
				return index;
			}
		}
```